sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -790,4 +791,15 @@ class APISpec extends StorageSpec {
             }
         }
     }
+
+    static String getAuthToken() {
+        if (env.testMode == TestMode.PLAYBACK) {
+            // we just need some string to satisfy SDK for playback mode. Recording framework handles this fine.
+            return Mono.just("recordingBearerToken")
```

Comment:
Wow yeah I missed that. This is wild. I'll fix.

Reply:
What worries me is that CI passed with this.

Reply:
Debugged. Groovy auto-calls `toString()` when needed and the method was declared to return `String`. The method was returning `"MonoJust"` in playback mode. Since the test just needed some string to function in playback, everything was fine.